modules/custom/az_card/src/Plugin/Field/FieldFormatter/AZCardDefaultFormatter.php

```diff
@@ -125,13 +125,20 @@ public function viewElements(FieldItemListInterface $items, $langcode) {
       // Link.
       $link_render_array = [];
       $link_url = '';
+      $current_path = \Drupal::service('path.current')->getPath();
       if ($item->link_title || $item->link_uri) {
         if (str_starts_with($item->link_uri ?? '', '/' . PublicStream::basePath())) {
           // Link to public file: use fromUri() to get the URL.
           $link_url = Url::fromUri(urldecode('base:' . $item->link_uri));
         }
         else {
-          $link_url = $this->pathValidator->getUrlIfValid($item->link_uri ?? '<none>');
+          // Check if the link is an anchor within the current page
```

Comment:
Adding a period to this comment should fix the PHPCS error:
```suggestion
          // Check if the link is an anchor within the current page.
```